tests/integration_tests.rs

```diff
@@ -360,7 +359,7 @@ async fn txn_bank_transfer() -> Result<()> {
             .await?;
         txn.put(bob.clone(), bob_balance.to_be_bytes().to_vec())
             .await?;
-        txn.commit().await?;
+        let _ = txn.commit().await;
```

Comment:
why ignore?

Reply:
Failure of a single transfer shouldn't affect the final invariant check. I think it's fine to revert the change now.